app/services/hawkular_proxy_service.rb

```diff
@@ -45,12 +47,14 @@ def get_data(id)
     starts = @params['starts'] || (ends - 8 * 60 * 60 * 1000)
     bucket_duration = @params['bucket_duration'] || nil
     order = @params['order'] || 'ASC'
-
-    client.gauges.get_data(id,
-                           :limit          => @params['limit'] || 100,
-                           :starts         => starts.to_i,
-                           :ends           => ends.to_i,
-                           :bucketDuration => bucket_duration,
-                           :order          => order)
+    limit = @params['limit'].to_i || 100
```

Comment:
We use the limit parameter when we want to get different number of answers, for example here:
 https://github.com/ManageIQ/manageiq/blob/master/app/assets/javascripts/controllers/container_live_dashboard/container_live_dashboard_controller.js#L102

we limit the number of last values to 5.